packages/sanes-chrome-extension/src/extension/background/model/backgroundscript/index.ts

```diff
@@ -87,6 +92,7 @@ class Backgroundscript {
     (window as IovWindowExtension).loadPersona = pss => this.loadPersona(pss);
     (window as IovWindowExtension).createAccount = () => this.createAccount();
     (window as IovWindowExtension).getPersonaData = () => this.getPersonaData();
+    (window as IovWindowExtension).hasStoredPersona = () => this.hasStoredPersona();
```

Comment:
I think it is better to select this type casting into separate variable like `windowExtension` or `iovWindowExtension`, shorter resulting code and easier to maintenance.

Reply:
I agree. Even better would be to explicitly pass a `target: IovWindowExtension` argument into this function instead of using the global reference